docs/user-guide/troubleshooting.rst

```diff
@@ -61,3 +61,38 @@ install`` tool is able to verify the server's SSL certificate with a trusted CA.
 Details on how to install a new trusted CA are outside the scope of this
 document.
 
+Missing metadata.json file
+--------------------------
+
+Symptom
+^^^^^^^
+
+Uploading a puppet module results in MissingModuleFile Exception raised.
```

Comment:
s/Uploading/If uploading/
Also combine this sentence with the sentence starting `One possible problem`...